docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -105,3 +105,27 @@ Runs  ``ansible localhost``- on your local system.
     "cache_updated": false,
     "changed": false
     #...
+
+ansible-doc
+===========
+
+* Show plugin names and their source files:
```

Comment:
Another friendly reminder from the DaWGs meeting that @acozine gave this comment:

I'd also use the same verb whenever possible, to make translation easier - `List available plugins` and `List plugin names` and their source files (or else `show` in both) as long as the two actions are the same (which they are in this case)

Reply:
Hi @oraNod, this is now done. Apologies for the looooong delay.